### DISCUSSION TITLE
tobqol sheesh alternative sound

### DISCUSSION THREAD

**Discussion by 69spacecowboy** [runelite/runelite#15270] [Ideas]
any possible way to get the old track of "sheeeesh" ( https://www.youtube.com/watch?v=YgT6XABqS5Y ) from spoon into tobqol?

**Comment by Nightfirecat** [comment]
Issues and suggestions for hub plugins must be made on that plugin's repository; they are not maintained by us. The repo is usually accessible from the Support link on the plugin.
